setup.py

```diff
@@ -0,0 +1,33 @@
+from setuptools import setup
+# from distutils.core import setup
+
+setup(
+    name = "plugit",
+    packages = ["plugit"],
+    version = "0.2.0rc1",
+    license = "BSD",
+    description = "PlugIt is a framework enhancing the portability and integration of web services requiring a user interface.",
+    author = "EBU Technology & Innovation",
+    author_email = "barroco@ebu.ch",
+    url = "http://github.com/ebu/PlugIt",
+    download_url = "",
+    keywords = ["frontend"],
+    classifiers = [
+        "Programming Language :: Python",
+        "Development Status :: 4 - Beta",
+        "Environment :: Web Environment",
+        "Framework :: Flask",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: OS Independent",
+        "Topic :: Software Development :: Libraries :: Python Modules",
+        "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+        ],
+    long_description = """\
```

Comment:
Many setup.py use the following contraption:

    open("README.md").read() + "\n\n"

See for example:
https://raw.githubusercontent.com/kennethreitz/requests/master/setup.py

As the long description will appear below the package in the pypi repository;
https://pypi.python.org/pypi/requests/2.7.0

It also remove a moving component where the doc need to be maintained.